examples/libf3d/CMakeLists.txt

```diff
@@ -1,14 +1,11 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.19)
 
-project(libf3d-examples)
+project(examples)
```

Comment:
im not sure we want to change the name of this project ? there is already a project on top called example